packages/@expo/cli/src/lint/lintAsync.ts

```diff
@@ -10,7 +10,7 @@ export const lintAsync = async (projectRoot: string) => {
 
   const manager = createForProject(projectRoot);
   try {
-    await manager.runAsync(['run', 'lint']);
+    await manager.runAsync(['run', 'eslint', '.']);
```

Comment:
This won't work for `npm` or `pnpm`. Here are the equivalents:
- `npm` -> `npx eslint .`
- `pnpm` -> `pnpm eslint .`

Alternatively, we could either spawn a new process or just `require('eslint/bin/eslint')` to keep it "working" for now.

Reply:
good call! wdyt about this? https://github.com/expo/expo/pull/28332/files#diff-674041de5e2d6dc1344c60e176ff815c71ad2c76c1ac823e2c99b77022813c4fR29